Descent3/multi_dll_mgr.cpp

```diff
@@ -595,7 +598,7 @@ int LoadMultiDLL(const char *name) {
   if (MultiDLLHandle.handle)
     FreeMultiDLL();
 
-  std::filesystem::path dll_path_name = std::filesystem::path(Base_directory) / "online";
+  std::filesystem::path dll_path_name = cf_GetWritableBaseDirectory() / "online";
```

Comment:
netcon modules can be placed not only in write-path.

Reply:
I agree, netcon modules can indeed be placed in read-only base directories. I’ve tested my code extensively, and I can confirm that my code works properly if you put the `online` directory inside one of the read-only base directories.

The line of code that you’re commenting on is part of a chunk of code that deletes temporary files. I think that it’s OK to delete temporary files that are in the writable base directory, but I don’t think that it’s OK to delete temporary files that are in any of the read-only base directories. It would be really confusing if we decided to call a list of directories “read-only” but then tried to write to those directories.

Reply:
Seems to be here old logic left after switching to use `Descent3_temp_directory`. This part of code now does nothing useful, so we can skip this request.